transform/schemas/properties/relations.py

```diff
@@ -0,0 +1,63 @@
+import os
+from schemas.properties.data import ID, LEGALNAME
+from schemas.properties.env import *
+from utils.loader import load_env_vars
+
+env_vars = load_env_vars()
+
+# HELPER COLUM NAMES
+DIRECTORY = "directory"
+FILE = "file"
+SOURCE = "source"
+SOURCE_FILE_PATH = "source_file_path"
+SOURCE_TYPE = "source_type"
+TARGET = "target"
+TARGET_FILE_PATH = "target_file_path"
+TARGET_TYPE = "target_type"
+
+# DICT KEYS
+DATASET_KEY = "dataset"
```

Comment:
Please add TODO: replace all DICT KEYS with pydantic settings - on development branch it should be handled by settings.DATASET... etc.

Reply:
added